src/Runtime/CMakeLists.txt

```diff
@@ -112,7 +112,7 @@ target_include_directories(PyRuntime
   )
 target_link_libraries(PyRuntime
   PRIVATE
-  ExecutionSession
+  OMExecutionSession
```

Comment:
I know another place using ExecutionSession that is https://github.com/onnx/onnx-mlir/blob/main/utils/RunONNXModel.py#L121. RunONNXModel.py is used by Jenkins to check the onnx model zoo.

Reply:
@tungld 

I have not renamed the module within the python interface because others may use it... I have tested this example

``` python
import numpy as np
from PyRuntime import ExecutionSession

# Load the model mnist.so compiled with onnx-mlir.
model = './mnist.so'

# Load the model mnist.so compiled with onnx-mlir.
model = './mnist.so'
session = ExecutionSession(model)
```

which is from mnist_runPyRuntime.py

Is that what the issue is?

Now if you agree that we should also change the module name inside python, to match the compiler interface...

Here is what I have (and like) for compiler module
``` python
import numpy as np
from PyCompileAndRuntime import OMCompileExecutionSession

# Load onnx model and create CompileExecutionSession object.
inputFileName = './mnist.onnx'
# Set the full name of compiled model
sharedLibPath = './mnist.so'
# Set the compile option as "-O3"
session = OMCompileExecutionSession(inputFileName,sharedLibPath,"-O3")

```

Reply:
@tungld let me know, will not merge before I get your input on this

Reply:
OK, so `ExecutionSession ` here is not the one in `from PyRuntime import ExecutionSession` . Then  it's OK to merge now. Thanks!

Reply:
```
Here is what I have (and like) for compiler module

import numpy as np
from PyCompileAndRuntime import OMCompileExecutionSession

# Load onnx model and create CompileExecutionSession object.
inputFileName = './mnist.onnx'
# Set the full name of compiled model
sharedLibPath = './mnist.so'
# Set the compile option as "-O3"
session = OMCompileExecutionSession(inputFileName,sharedLibPath,"-O3")
```

I see we can change  the python module from `ExecutionSession` to `OMExecutionSession` to make it consistent with `OMCompileExecutionSession`. Totally agree.

Reply:
Renamed ExecutionSession to OMExecutionSession in the python object. Believe that I made the changes everywhere.

Reply:
Thanks so much! LGTM!